app/connectors/aws/SqsQueueConsumer.scala

```diff
@@ -17,12 +17,11 @@
 package connectors.aws
 
 import javax.inject.Inject
-
 import com.amazonaws.services.sqs.AmazonSQS
 import com.amazonaws.services.sqs.model.{DeleteMessageRequest, ReceiveMessageRequest, ReceiveMessageResult}
 import config.ServiceConfiguration
 import model.Message
-import services.QueueConsumer
+import services.{QuarantineQueueConsumer, QueueConsumer, SuccessfulQueueConsumer}
```

Comment:
The connector package should be generic/reusable. It shouldn't depend upon business-level services. This dependency between the packages is backwards, and possibly cyclic.